curvefs/src/metaserver/dentry_manager.cpp

```diff
@@ -20,79 +20,77 @@
  * Author: chenwei
  */
 
-#include "curvefs/src/metaserver/dentry_manager.h"
 #include <glog/logging.h>
 
+#include <sstream>
+
+#include "curvefs/src/metaserver/dentry_manager.h"
+
 namespace curvefs {
 namespace metaserver {
-MetaStatusCode DentryManager::CreateDentry(const Dentry& dentry) {
-    VLOG(1) << "CreateDentry, dentry: " << dentry.ShortDebugString();
-    MetaStatusCode status = dentryStorage_->Insert(dentry);
-    if (status != MetaStatusCode::OK) {
-        LOG(ERROR) << "CreateDentry fail, dentry: " << dentry.ShortDebugString()
-                   << ", ret = " << MetaStatusCode_Name(status);
-        return status;
-    }
 
-    VLOG(1) << "CreateDentry success, dentry: " << dentry.ShortDebugString();
-    return MetaStatusCode::OK;
+DentryManager::DentryManager(std::shared_ptr<DentryStorage> dentryStorage,
+                             std::shared_ptr<TxManager> txManager)
+    : dentryStorage_(dentryStorage),
+      txManager_(txManager) {}
+
+void DentryManager::Log(const std::string& request, const Dentry& dentry) {
```

Comment:
It is not a good idea using a function like this, beacuse it will sacrifice readability

Reply:
> It is not a good idea using a function like this, beacuse it will sacrifice readability

I will improve it.